tests/AppendStreamTest.php

```diff
@@ -20,23 +16,21 @@ public function testValidatesStreamsAreReadable()
         $s->expects($this->once())
             ->method('isReadable')
             ->will($this->returnValue(false));
+
+        $this->expectExceptionGuzzle('InvalidArgumentException', 'Each stream must be readable');
```

Comment:
Wouldn't it make sense to use `::class` constants here? I kind of like seeing all the references to a class if possible.

Reply:
No, because we support PHP 5.4.